api/apps/api/src/modules/projects/projects-root.service.ts

```diff
@@ -0,0 +1,67 @@
+import { Injectable } from '@nestjs/common';
+import { FetchSpecification } from 'nestjs-base-service';
+import { AppInfoDTO } from '../../dto/info.dto';
+
+import { GeoFeaturesService } from '../geo-features/geo-features.service';
+
+import { ProjectsCrud } from './projects-crud';
+import { ProtectedAreasFacade } from './protected-areas/protected-areas.facade';
+import { Project } from './project.api.entity';
+import { CreateProjectDTO } from './dto/create.project.dto';
+import { UpdateProjectDTO } from './dto/update.project.dto';
+
+@Injectable()
+export class ProjectsRootService {
```

Comment:
again naming/intent - if this should be come the root/demigod service for Projects, I'd maybe just call it `ProjectsService` 🤔

Reply:
though in practice it seems to be a mix of mapper + facade, if I understand the intent?

Reply:
ok, I was looking at an earlier commit - agree with keeping the name as `ProjectsService` ofc 😄

Reply:
👍 started sweating I missed this 🕺

Reply:
@hotzevzl the only reason it was named `-root` in first commit is to have a nice diff, otherwise (as in general diff) it was  messed up with new implementation - and thus hard to read.

Reply:
yep, sorry about the noise 😅